h3/src/proto/frame.rs

```diff
@@ -351,6 +351,17 @@ impl SettingId {
         )
     }
 
+    /// Returns if a Settings Identifier is forbidden
+    fn is_forbidden(&self) -> bool {
+        //= ci/compliance/specs/rfc9114.txt#7.2.4.1
+        //# their receipt MUST be treated as a connection error of type
+        //# H3_SETTINGS_ERROR.
```

Comment:
I would consider quoting the whole paragraph to give context:

```suggestion
        //# Setting identifiers that were defined in [HTTP/2] where there is no
        //# corresponding HTTP/3 setting have also been reserved
        //# (Section 11.2.2).  These reserved settings MUST NOT be sent, and
        //# their receipt MUST be treated as a connection error of type
        //# H3_SETTINGS_ERROR.
```